### PR TITLE
describe server extension suite

### PR DESCRIPTION
**Pull request by mcfnord**
This PR adds coverage of server extension tools [described in this discussion](https://github.com/orgs/jamulussoftware/discussions/3085). The tools all work together, so I cover them together.

### CODE REVIEW

File: docs/RELATED-PROJECTS.md [hunk 1]
```diff
@@ -49,3 +49,14 @@ within a Docker container. It provides a `Dockerfile` and some documentation.
...
+## Server add-on suite
+
+[These components](https://github.com/orgs/jamulussoftware/discussions/3085) extend the features of a Jamulus server.
+
+* [jamulus-lounge](https://github.com/dtinth/jamulus-lounge) adds a web-based interface that lets
+people listen to a Jamulus server. It connects to a server using a bot named **lobby**.
+* [gojam](https://github.com/dtinth/gojam) is an implementation of a Jamulus client written in Go.
+This tool is intended for headless streaming use-cases. Used by **jamulus-lounge**.
+* [jamviz](https://github.com/dtinth/jamviz) shows active connections of a Jamulus server that
+is running **jamulus-lounge**. Used to show activity to livestreaming audience.
```
Comment (mcfnord) [inline docs/RELATED-PROJECTS.md:62]:
```suggestion
is running **jamulus-lounge**. Used to show activity to a livestreaming audience.
```

**Comment by mcfnord** [comment]
I'd remove the datestamp to Tony's byline. The datestamp might cause some to think this document is outdated. I'd prefer no datestamp on a document of this kind. Is this a code change I can make on this branch somehow?

**Comment by ann0see** [comment]
If you're talking about the website - Maybe the framework we're using - Jekyll can change the date in the URL. (see _config.yml).

**Comment by mcfnord** [comment]
Yes, [it renders like this](https://jamulus.io/kb/2022/01/25/Related-Projects.html). I guess [the datestamp] (https://github.com/jamulussoftware/jamuluswebsite/blob/release/_config.yml#L37) seems pretty essential to this architecture. It's not in-file metadata like the author is. 

Here's a big uh-oh: [What file is this](https://github.com/jamulussoftware/jamuluswebsite/blob/release/_posts/2022-01-25-Related-Projects.md)? Is this a necessary duplication? Am I modifying the wrong file?

**Comment by ann0see** [comment]
Uh Yes. You're modifying the wrong file. I think there was an issue about merging both? Not sure what the outcome was. But probably we should remove this one.

**Comment by mcfnord** [comment]
Trying https://github.com/jamulussoftware/jamuluswebsite/pull/932 instead.

**Comment by mcfnord** [comment]
Also trying to remove this particular duplication at https://github.com/jamulussoftware/jamulus/pull/3091.

**Pull request closed without merging**